lib/instr.perf.ml

```diff
@@ -0,0 +1,17 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+let read_counter : unit -> (Int64.t, string) Result.t =
+  let drop () = try Perf.drop () with Failure(_) -> () in
+  try
+    Perf.init (); at_exit drop;
```

Comment:
There is not much point dropping at exit is there? The OS can figure it out when it cleans up the process.

Reply:
Yeah, in that particular case we'd be fine, but I generally like to explicitly release resources, especially if they are allocated via C code.